docs/meshcloud.policies.md

```diff
@@ -41,17 +82,27 @@ You may encounter policies in the meshPanel when doing any of the following acti
 
 It might happen, consciously or unconsciously, that you violate one or more policies. At every place in the meshPanel that is mentioned above this section, we prevent you from finalizing the violation and you will be prompted with an error message explaining which meshPolicy or policies you violated, and why.
 
-Take this policy violation as an example (see the picture below), where we have a meshPolicy defined on 'meshCustomer' and 'meshProject', both on the `environment` tag.
+Take this meshPolicy violation as an example (see the picture below), where we have a meshPolicy defined on 'meshCustomer' and 'meshProject' with the evaluation strategy 'Subset', both on the `environment` tag.
 
 ![Example Policy Error Message](assets/mesh_policies/policies_example_error_message.png)
 
 The project that is being created called 'my-example-project-prod' wants environment `prod` and the meshCustomer 'managed-customer' the project is being created in, has environments `dev`, `test` and `qa` defined. This means there is a mismatch as `<prod>` is not inside `<dev, test, qa>`. In order to solve this problem, we have to pick an environment that is defined on the meshCustomer, e.g. `dev`. After picking a valid environment value, we can save the project again, and (if we don't violate any further policies) the project is successfully created, including the right compliance for your organization! ✅
 
 ### Are there any other places where policies are enforced?
 
-Besides end-users being impacted at the places above, there are also other places where policy violations could be caused. The diagram below describes all possible relationships between meshObjects and the behavior that is expected, depending on who is doing the change.
+Besides end-users being impacted at the places above, there are also other places where meshPolicy violations could be caused. The diagram below describes all possible relationships between meshSubjects and the behavior that is expected, depending on who is doing the change.
 
-![Policy Relationships](assets/mesh_policies/policy_relationships.png)
+```mermaid
```

Comment:
I find this diagram a little bit messy but I also do not really have any better proposals.

Isn't it the case though that the directions of these arrows have to do with the authoritative / affected mechanism? Maybe that explains it better than this diagram?

Reply:
We want to show whats the meaning is between the subjects independent of authoritative and affected.
We've decided within the proposal to use mermaid. So I try to implement the diagram depending on the notion guide https://www.notion.so/meshcloud/meshcloud-docs-diagrams-with-Mermaid-JS-c3deaea4ce8f40ad95fdde1a56b678e0 .